README.md

```diff
@@ -38,6 +38,8 @@ install.sh
 Makefile.am
 include/bitcoin/[library].hpp
 include/bitcoin/[library]/version.hpp
+builds/cmake/CMakeLists.txt
+builds/cmake/modules/[module].cmake
```

Comment:
This should probably be [library].

Reply:
That depends on how to define the "[library]". If we assume [library] is a any library then you are right. If we assume [library] is the name of the current repository or any libbitcoin library then you are wrong. 

For example the libbitcoin repository contains following modules
https://github.com/libbitcoin/libbitcoin/tree/master/builds/cmake/modules

Reply:
Yeah, in this case [dependency] would be better.